pkg/controllers/management/podsecuritypolicy/template.go

```diff
@@ -0,0 +1,50 @@
+package podsecuritypolicy
+
+import (
+	"fmt"
+
+	"github.com/rancher/types/apis/management.cattle.io/v3"
+	"github.com/rancher/types/config"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+func Register(management *config.ManagementContext) {
+	lifecycle := &lifecycle{
+		psptpbs:      management.Management.PodSecurityPolicyTemplateProjectBindings(""),
+		psptpbLister: management.Management.PodSecurityPolicyTemplateProjectBindings("").Controller().Lister(),
+	}
+
+	management.Management.PodSecurityPolicyTemplates("").AddLifecycle("mgmt-pspt-lfc-handler", lifecycle)
+}
+
+type lifecycle struct {
+	psptpbs      v3.PodSecurityPolicyTemplateProjectBindingInterface
+	psptpbLister v3.PodSecurityPolicyTemplateProjectBindingLister
+}
+
+func (l *lifecycle) Create(obj *v3.PodSecurityPolicyTemplate) (*v3.PodSecurityPolicyTemplate, error) {
+	return obj, nil
+}
+
+func (l *lifecycle) Updated(obj *v3.PodSecurityPolicyTemplate) (*v3.PodSecurityPolicyTemplate, error) {
```

Comment:
This will need an impl to update the actual PSP when the PSPT changes (maybe you just havent gotten to it yet)

Reply:
No! IM wrong. this is the management side controller. my bad